commands/host.vegman

```diff
@@ -98,12 +98,11 @@ function cmd_power_off {
   fi
 
   if [[ -n "${force}" ]]; then
-    ipmitool chassis power off
-    echo "Chassis power is switched off, host is down."
+    ipmitool power off
   else
     ipmitool power soft
-    echo "Shutdown signal sent to host, waiting to power the system off."
   fi
+  echo "Shutdown signal sent to host, waiting to power the system off."
```

Comment:
`ipmitool power off` will turn the host off an any case. Why "waiting"?

Reply:
Yup. Besides, nothing in the code is actually 'waiting'.